flask/todo_app/models/user_models.py

```diff
@@ -47,7 +28,4 @@ class Role(db.Model):
 
     @classmethod
     def get_all_for_user_id(cls, user_id):
```

Comment:
all these classmethods that are doing queries in models should be removed.  I can clean that up after we get it merged.